actions.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+#
+
+register-user:
+  description: Registers a user for the Synapse server.
+  properties:
+    username:
+      description: User name
```

Comment:
Would it be worth it to make these descriptions be a bit more informative? They are essentially re-stating the name at the moment

Reply:
also for `password`

Reply:
Added some more information for the fields.